hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1433,18 +1452,39 @@ func reconcileTuningConfigMap(tuningConfigMap *corev1.ConfigMap, nodePool *hyper
 		tuningConfigMap.Labels = make(map[string]string)
 	}
 
-	tuningConfigMap.Labels[tuningConfigMapLabel] = "true"
-	tuningConfigMap.Annotations[nodePoolAnnotation] = nodePool.GetName()
+	tuningConfigMap.Annotations[nodePoolAnnotation] = client.ObjectKeyFromObject(nodePool).String()
```

Comment:
why this change? was it broken before for tuningConfigMap reconciliation?

fwiw this would give a particular hcp the ability to reconcile nodepools for others HCs by applying an annotation. We do the same for other resources but I rather not perpetuate it if possible.

Reply:
I'm sorry, still don't get the reason for this particular change. Before this PR the nonodePoolAnnotation was storing the name
`tuningConfigMap.Annotations[nodePoolAnnotation] = nodePool.GetName()`
now is storing the namespaced name:
`tuningConfigMap.Annotations[nodePoolAnnotation] = client.ObjectKeyFromObject(nodePool).String()`

Reply:
it was an error.
As you can see [here](https://github.com/openshift/cluster-node-tuning-operator/blob/16ca57f9a7ac302bc07dc7c9f0fd91140da97277/pkg/operator/hypershift.go#L146) this annotation is used in NTO to read the namespacedname of the NodePool